### ISSUE TITLE
sfinv

### ISSUE THREAD

**Issue by ghost** [minetest-mods/character_creator#12]
I want to add the supporting of sfinv to this. If skinsdb is available, don't need to add a tab to sfinv.

**Comment by bell07** [comment]
Supported indirect trough skinsdb integration. In skinsdb a character creator skin appears and starts character creator on selection. Each player can decide self if he uses character_creator or a skin

**Comment by ghost** [comment]
@bell07 Would you maintain this mod instead of me? I have no time to work for the mods nowadays...
P.S. I added you to this repository as admin for now.

**Comment by bell07** [comment]
@Rui-Minetest, I accepted your invitation. But I am away from Minetest too and just maintain some reported bugs or review PR's if I find time. See https://github.com/minetest-mods/minetest-mods.github.io/issues/74. I do not play the game anymore.
Unsure if the right way is to open new issue on minetest-mods/minetest-mods.github.io to ask for new maintainer

**Comment by ghost** [comment]
I see. After opening a new issue, I'm going to close my github account once the transferring is complete.
https://github.com/minetest-mods/minetest-mods.github.io/issues/96
